Aptoide/aptoide-client-v8
# [APP-27] feature: show beta icon on app view for alpha, beta apps.

**What does this PR do?**

   Show beta icon on App View for alpha and beta apps.

**Database changed?**

   No

**Where should the reviewer start?**


**How should this be manually tested?**

  Using search find beta apps and check if they are showing the new icon.
  Validate if other apps are not showing the new icon.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-27](https://aptoide.atlassian.net/browse/APP-27)

**Code Review Checklist**
